modules/imgproc/src/opencl/box_filter.cl

```diff
@@ -0,0 +1,78 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+__kernel void box_filter(__global const uint* src, int ssy, int srcOffsetX, int srcOffsetY,
```

Comment:
yes, src and dst offset (assume 0) are used in the kernel, I am considering remove them from ocl kernel and check them in the host code.